requirements.txt

```diff
@@ -2,6 +2,9 @@ civis~=1.6
 requests~=2.18
 click~=6.7
 notebook~=5.1,!=5.7.*
+# for versions of jupyter before 5.7.5 we need to specify tornado<6
+# as these older versions of jupyter will break with tornado 6
+tornado<6
```

Comment:
I think we should upgrade the notebook package to something like `notebook>=5.7.5,<6.0.0`

I can't remember why we do not want any `5.7` versions ... 

If we can successfully upgrade the notebook version to `5.7.x` then I think we can also fix the issue with upgrading python3 docker image to use datascience-python 5.x

Reply:
#33

Reply:
I don't really have bandwidth to investigate the impact of changing our jupyter version. I was just trying to fix what we have currently.